packages/samples/mvp-dapp/src/main.ts

```diff
@@ -1,15 +1,35 @@
 import Koa from 'koa'
 import { koaBody } from 'koa-body'
 import { initialKuai } from '@ckb-js/kuai-core'
-import { KoaRouterAdapter, CoR } from '@ckb-js/kuai-io'
+import { KoaRouterAdapter, CoR, TipHeaderListener } from '@ckb-js/kuai-io'
 import { router } from './app.controller'
 import './type-extends'
+import { Manager, ProviderKey } from '@ckb-js/kuai-models'
+import { NervosChainSource } from './chain-source'
 
 async function bootstrap() {
   const kuaiCtx = await initialKuai()
   const kuaiEnv = kuaiCtx.getRuntimeEnvironment()
   const port = kuaiEnv.config?.port || 3000
 
+  Reflect.defineMetadata(
+    ProviderKey.Actor,
+    {
+      ref: {
+        name: 'resource',
+        path: '',
+        protocal: 'local',
+        uri: 'local://resource',
+      },
+    },
+    Manager,
+  )
+
+  const dataSource = new NervosChainSource(kuaiEnv.config.rpcUrl)
+  const listener = new TipHeaderListener(dataSource)
+  const manager = new Manager(listener, dataSource)
+  manager.listen()
```

Comment:
The `manager` should export for other modules.

Reply:
What I design is to interact with resource binding manager by message bus.

Reply:
Will it bind in the `Register`? Will exist some manager instances or only one instance in the DApp process?

Reply:
> Will it bind in the `Register`? Will exist some manager instances or only one instance in the DApp process?

It will bind in `Register`, and I think there will be only one manager instance in a DApp.